prql-php/src/MessageKind.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Prql\Compiler;
+
+/**
+ * Compile message kind. Currently only Error is implemented.
+ */
+enum MessageKind
+{
+    case Error;
+    case Warning;
+    case Lint;
```

Comment:
Should this one be called "Lint" or "Information" ?

Reply:
It's called lint in prql-compiler. I think it's [appropriate name](https://en.wikipedia.org/wiki/Lint_(software)).

Reply:
Visual Studio uses the word "Messages".
![image](https://user-images.githubusercontent.com/10222521/224075264-9417e0af-61f1-465e-ba95-14702644f70b.png)